boards/espressif/esp32c3_devkitc/Kconfig.defconfig

```
@@ -0,0 +1,11 @@
+# ESP32C3 devkitc board configuration
+
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+config HEAP_MEM_POOL_ADD_SIZE_BOARD
+	int
+	default 65535 if WIFI && BT
```

fwiw other esp32 boards have recently been updated to:

```suggestion
	default $(UINT16_MAX) if WIFI && BT
```

not sure this was fully intended though, as it looks to me as if this is a reasonably arbitrary figure that just happens to be equal to UINT16_MAX, not really something that's semantically *meant* to be UINT16_MAX

either way, it should probably be consistent across esp32 boards

---

A more suitable macro here would be `KB(64)`   :)

---

@kartben Regarding the UINT16_MAX in those Kconfig, I think it is totally wrong. I approved that change though. I might submit a PR to restore it to `65535` or KB(64). This value has nothing to do with `UINT16_MAX` at all.

---

@sylvioalves I agree it should be reverted -- and I should point out I also missed it during review :)